packages/perseus/src/types.ts

```diff
@@ -395,6 +402,24 @@ export type Tracking =
     // Track all interactions
     | "all";
 
+// See graded-group widget
+export type TrackingGradedGroupExtraArguments = {
+    status: "correct" | "incorrect" | "invalid";
+};
+
+// See sequence widget
+export type TrackingSequenceExtraArguments = {
+    visible: number;
+};
+
+// Add extra widget tracking arg types into this type as needed.
+// NOTE(jeremy): I've added `Empty` at the start to make explicity that many
+// widgets don't add extra arguments. It also has the nice side-effect of
+// formatting more nicely. :)
+export type TrackingExtraArguments = Empty &
+    TrackingGradedGroupExtraArguments &
+    TrackingSequenceExtraArguments;
```

Comment:
I'd probably wouldn't even bother with this type.  I think doing the following where `TrackInteractionArgs` is defined would better communicate intent:
```
type TrackInteractionArgs = {
   ...
} & Partial<TrackingGradedGroupExtraArguments> 
  & Partial<TrackingSequenceExtraArguments>
```